README.md

```diff
@@ -50,3 +50,22 @@ Note: _(One must have issued `npm install` previously.)_
 Run e.g. on master: `mvn -B release:prepare` e.g. via [![Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io#https://github.com/baloise/corellia)
 
 Subsequently the GitHub action worksflow "create release" will pick up the published tag and release and deploy the artifacts in the Github package registry.
+
+
+## Spectral linting (just to not forget how to do it.)
+
+use the following comand to lint openapi.json
+```
+node_modules\.bin\spectral.cmd lint docs\openapi.json
```

Comment:
Use npm with `npm run spectral`. We should not access `node_modules` directly.

Reply:
you are right, done in the github action but forgot in the md file